flash/vision/classification/data.py

```diff
@@ -527,3 +534,46 @@ def from_filepaths(
             seed=seed,
             **kwargs
         )
+
+
+class _CustomBaseVisualization(BaseVisualization):
```

Comment:
Make this one the default for `ImageClassificationData` directly.

Reply:
done